tests/helper_test.go

```diff
@@ -266,3 +266,46 @@ func (tu *testUtils) AssertEval(t *testing.T, page playwright.Page, script strin
 	require.NoError(t, err)
 	require.Equal(t, expected, result)
 }
+
+func TestConvertSelectOptionSet(t *testing.T) {
+	testCases := []struct {
+		name         string
+		optionValues playwright.SelectOptionValues
+		expected     interface{}
+	}{
+		{
+			name:         "SelectOptionValues is nil",
+			optionValues: playwright.SelectOptionValues{},
+			expected:     make(map[string]interface{}),
+		},
+		{
+			name: "SelectOptionValues is supplied",
+			optionValues: playwright.SelectOptionValues{
+				Value: &[]string{"a", "b"},
```

Comment:
Maybe we should add helper functions as we have for String(), Int() for a StringSlice and IntSlice. What do you think?

Reply:
Good idea. Will add the functions.